tbx/project_styleguide/templates/patterns/base.html

```diff
@@ -21,14 +21,14 @@
             <meta name="robots" content="noindex">
         {% endif %}
 
-        {% block meta_tags %}{% endblock %}
+        {% comment %} {% block meta_tags %}{% endblock %} {% endcomment %}
```

Comment:
Question: Is this temporary?

Reply:
oh that may be a mistake from when I was debugging lots of issues in the pattern library, which turned out to be some general settings that needed updating.

Reply:
Comments removed, and I spotted the yaml file for `base_page.html` was in the wrong folder, so this is also fixed.